eclipse-collections/src/main/java/org/eclipse/collections/impl/bimap/mutable/AbstractMutableBiMap.java

```diff
@@ -616,6 +618,26 @@ public void forEachValue(Procedure<? super V> procedure)
         return this.delegate.aggregateInPlaceBy(groupBy, zeroValueFactory, mutatingAggregator);
     }
 
+    public <V1> MutableObjectLongMap<V1> sumByInt(Function<? super V, ? extends V1> groupBy, IntFunction<? super V> function)
+    {
+        return this.getDelegate().sumByInt(groupBy, function);
+    }
+
+    public <V1> MutableObjectDoubleMap<V1> sumByFloat(Function<? super V, ? extends V1> groupBy, FloatFunction<? super V> function)
+    {
+        return this.getDelegate().sumByFloat(groupBy, function);
+    }
+
+    public <V1> MutableObjectLongMap<V1> sumByLong(Function<? super V, ? extends V1> groupBy, LongFunction<? super V> function)
+    {
+        return this.getDelegate().sumByLong(groupBy, function);
+    }
+
+    public <V1> MutableObjectDoubleMap<V1> sumByDouble(Function<? super V, ? extends V1> groupBy, DoubleFunction<? super V> function)
+    {
+        return this.getDelegate().sumByDouble(groupBy, function);
+    }
+
```

Comment:
Similarly, you can call `this.delegate` instead of `this.getDelegate()` in these 4 methods.